profile/prune.go

```diff
@@ -22,6 +22,37 @@ import (
 	"strings"
 )
 
+var (
+	reservedNames = []string{"(anonymous namespace)", "operator()"}
+	bracketRx     = regexp.MustCompile("(" + strings.Replace(strings.Replace(
```

Comment:
You might write:
```golang
	bracketRx     = func() *regexp.Regexp {
		var quotedNames []string
		for _, name := range append(reservedNames, "(") {
			quotedNames = append(quotedNames, regexp.QuoteMeta(name))
		}
		return regexp.MustCompile(strings.Join(quotedNames, "|"))
	}()
```

The opening and closing parentheses you wrap the regexp in are also unnecessary, I believe (so I've omitted them above).